team/index.html

```diff
@@ -125,7 +125,7 @@ <h2>Administrators</h2>
 
   <section class="row">
 
-    <h2>Community Moderators</h2>
+    <h2 id="triage">Community Moderators</h2>
```

Comment:
Can we name the target `moderators` like the title?
```suggestion
    <h2 id="moderators">Community Moderators</h2>
```